src/test/Components/CreateImageWizard/steps/FileSystemConfiguration/FileSystemConfiguration.test.tsx

```diff
@@ -221,6 +222,7 @@ describe('Step File system configuration', () => {
     await selectGuestImage();
     await goToFileSystemConfigurationStep();
     await goToReviewStep();
+    await openAndDismissSaveAndBuildModal();
```

Comment:
This seems really strange to me. I'd like to understand why we need to call this function in some tests, and remove it from other tests. I don't see how adding a new user step should have any impact on whether or not the save and build modal appears and needs to be dismissed. :thinking:

Reply:
The `openAndDismissSaveAndBuildModal()` function should be called only once per test file, always in the first test that clicks on the "Create blueprint" button. When the popup gets dismissed, it won't appear again. We store something like "popupSeen" in the local storage. This should persist throughout test file.

That being said - I've tried to revert the changes and the tests finished perfectly fine. What problem was it causing @mgold1234? I mean it's functionally the same, this way we just click on "Create blueprint" and dismiss the popup "prematurely" as we don't interact with the create button further in "revisit step button on Review works".